giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/service/reprocessing/IReprocessingService.java

```diff
@@ -3,5 +3,9 @@
 import edu.asu.diging.gilesecosystem.web.core.model.IDocument;
 
 public interface IReprocessingService {
+    /**
+    * Initiates the reprocessing of the specified document.
```

Comment:
This needs to be a little more detailed. Which components are run again? What happens to the already extracted files, etc?